scriptworker/test/__init__.py

```diff
@@ -241,6 +242,14 @@ def integration_create_task_payload(config, task_group_id, scopes=None,
     }
 
 
+@contextmanager
+def working_directory(path):
```

Comment:
This isn't in the `pytest` base package. and that makes me :thinking:: is this not the right way of doing this? Let me know if there's a better way doing this :)

Reply:
Nice! that's exactly what I was looking for :)